torchao/prototype/low_bit_optim/subclass_fp8.py

```diff
@@ -11,13 +12,26 @@
 DTYPE = torch.float8_e4m3fn
 
 
-def quantize_fp8(input: Tensor, block_size: int):
+def quantize_fp8(input: Tensor, block_size: int, dynamic_range_expansion: bool):
+
     shape = input.shape
     input = input.view(-1, block_size)
+    k =  aten.ones(input.shape[0], dtype=DTYPE, device=input.device) if dynamic_range_expansion else None
```

Comment:
You can just set `k = None` here. It will be overwritten later if `dynamic_range_expansion=True`

Reply:
okay will do it